CHANGES.md

```diff
@@ -1,5 +1,24 @@
 # Changes
 
+## edge-23.2.1
+
+This edge release sees the `linkerd-cni` plugin moved to
+`linkerd2-proxy-init` and released from that repository. An iptables
+improvement to `linkerd-cni` and `proxy-init` is the main focus. Other
+minor fixes are also included.
+
+* Upgrades `linkerd-cni` to v1.0.0 which is now published from
+  `linkerd2-proxy-init` and makes iptables rules idempotent (thanks @jim-minter!)
+* Upgraded `proxy-init` from v2.1.0 to v2.2.0 which makes `iptables` rules
+  idempotent (thanks @jim-minter!)
```

Comment:
I believe logging was also improved for `linkerd2-proxy-init` and `linkerd-cni`?

Reply:
yes for `linkerd-cni` but not for `linkerd2-proxy-init`. Added!